app/policies/quickbooks_policy.rb

```diff
@@ -0,0 +1,13 @@
+class QuickbooksPolicy < ApplicationPolicy
```

Comment:
Also should be namespaced under `Accounting`. `Admin` is not required here as that's for controllers only.